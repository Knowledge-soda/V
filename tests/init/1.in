f(x, y) := *(x, +(x, y))
:a, c
=(c, read())
=(a, v(c))
print(f(v(a), v(c)))
