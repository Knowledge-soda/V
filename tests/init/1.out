_start:
    NAME(id) BEGIN NAME(=) BEGIN NAME(c) SEP NAME(read) BEGIN END END NAME(=) BEGIN NAME(a) SEP NAME(v) BEGIN NAME(c) END END NAME(print) BEGIN NAME(f) BEGIN NAME(v) BEGIN NAME(a) END SEP NAME(v) BEGIN NAME(c) END END END END
f:
    BEGIN NAME(x) SEP NAME(y) END : NAME(*) BEGIN NAME(x) SEP NAME(+) BEGIN NAME(x) SEP NAME(y) END END
